.github/workflows/integration.yml

```diff
@@ -76,7 +76,7 @@ jobs:
         run: |
           tag=${GITHUB_REF##*/}
           sed --in-place "s/APP_VERSION_PLACEHOLDER/$tag/g" chart/otomi/Chart.yaml
-          sed --in-place "s/CHART_PATCH_PLACEHOLDER/0/g" chart/otomi/Chart.yaml
+          sed --in-place "s/0-chart-patch-placeholder/0/g" chart/otomi/Chart.yaml
```

Comment:
can't it just be `chart-patch-placeholder`?

Reply:
no, because 0.0-chart-patch-placeholder is not semver compatible
0.0.0-chart-patch-placeholder is, so you can install otomi form local chart